manifests/apache.pp

```diff
@@ -26,15 +26,31 @@
 
   include ::apache::mod::headers
   include ::apache::mod::rewrite
+  include ::apache::mod::ssl
   apache::listen { $pimcore::params::port: }
 
-  $epp_config = { 'app_name' => $pimcore::app_name }
-  apache::vhost::custom { 'pimcore':
-    content => epp('pimcore/apache2.conf.epp', $epp_config)
-  }->
   file { "/var/www/html/${pimcore::app_name}":
     ensure  => 'link',
     target  => "/opt/pimcore/${pimcore::app_name}/public",
   }
+
+  apache::vhost { 'pimcore':
+    ssl         => true,
+    ssl_cert    => $pimcore::ssl_cert,
```

Comment:
A good habit is to use $::varname::something instead of $varname::something when referring to fully-qualified variable paths. In certain edge cases you might get interesting behavior with $varname::something.